tests/mocha/client/sampleClientTest.js

```diff
@@ -0,0 +1,11 @@
+// Sample test generated by Velocity for reference
```

Comment:
Definitely not for long. It does look idiotic.

I like to have them around until I/we understand how to write these tests better. For the client version, it also keeps the "magic" client directory from being pruned by git.

I'm happy to just keep them on my machine if you prefer that.